### DISCUSSION TITLE
Specimen_procedure_type vs procedure_type

### DISCUSSION THREAD

**Discussion by saskiad** [AllenNeuralDynamics/aind-data-schema#618] [General]
In the specimen procedures, we already had a procedure type, and the new specimen_procedure_type is adding not useful information. Is it possible to rely on the existing procedure_type and remove the specimen_procedure_type? @jtyoung84

**Comment by jtyoung84** [comment]
> In the specimen procedures, we already had a procedure type, and the new specimen_procedure_type is adding not useful information. Is it possible to rely on the existing procedure_type and remove the specimen_procedure_type? @jtyoung84

Yeah, the only caveat is that we'll need one field that is a Literal string that's unique to each model if we want the user to be able to select from a list of Procedures.

**Reply by saskiad** [reply]
I believe that's what procedure_type is, unless I am misunderstanding something.

This is what this looks like currently. It's clunky having so many names and types for each procedures, especially where in some situations the specimen_procedure_type is specific to the procedure (Immunlabeling) while in others it says "Specimen Procedure". The procedure_type uses an enum so doesn't that meet the Literal criteria?

"specimen_procedures": [
    {
        "specimen_procedure_type": "Specimen Procedure",
        "specimen_id": "609281",
        "procedure_name": "DCM Delipidation",
        "procedure_type": "Delipidation",
...
    },
    {
        "specimen_procedure_type": "Specimen Procedure",
        "specimen_id": "609281",
        "procedure_name": "SBiP Delipidation",
        "procedure_type": "Delipidation",
...
    },
    {
        "specimen_procedure_type": "Immunolabeling",
        "specimen_id": "609281",
        "procedure_name": "Immunostaining primary",
        "procedure_type": "Immunostaining",
    ...
    },
    {
        "specimen_procedure_type": "Immunolabeling",
        "specimen_id": "609281",
        "procedure_name": "Immunostaining secondary",
        "procedure_type": "Immunostaining",
...
    },
    {
        "specimen_procedure_type": "Specimen Procedure",
        "specimen_id": "609281",
        "procedure_name": "Gelation",
        "procedure_type": "Gelation",
...
    },
]

**Reply by jtyoung84** [reply]
That should work. We can use the "procedure_type" field.

**Comment by jtyoung84** [comment]
This should be addressed in this PR: https://github.com/AllenNeuralDynamics/aind-data-schema/pull/667
